### PR TITLE
JEP-14 - Update the roadmap process

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Updates the roadmap JEP to reflect the current roadmap page state and data structure. Also includes unmerged changes from https://github.com/jenkins-infra/jenkins.io/pull/3523

### CODE REVIEW

File: jep/14/README.adoc [hunk 1]
```diff
@@ -83,70 +83,101 @@ Current version of the roadmap can be found here: link:https://www.jenkins.io/pr
...
-* **Near Term** - a wider area of focus with more flexibility of
-scope and design options (no specific timeframe but will be reconsidered
-in the future)
-* **Future** - higher level things to be considered with a broad scope
- (timescales varied - later items will be reprioritized once _Current_ and _Near Term_ items progress significantly)
-* **Completed** - Initiative which is largely completed and, in the case of user-facing changes, released/deployed.
+Instead of providing delivery dates for various initiatives on the roadmap, we use the following "horizons":
+
+* **Future** - There is a consensus in the community that we would like this initiative to happen.
+  We intend to work on that in the future, but there is no ongoing development.
```
Comment (slide) [inline jep/14/README.adoc:89]:
```suggestion
  We intend to work on that in the future, but there is no active development.
```

File: jep/14/README.adoc [hunk 2]
```diff
@@ -83,70 +83,101 @@ Current version of the roadmap can be found here: link:https://www.jenkins.io/pr
...
+* **Near Term** - We intend to start working on this initiative in the short term.
+  There are contributors who declared their plans to work on this initiative soon, but there is no ongoing development.
```
Comment (slide) [inline jep/14/README.adoc:91]:
```suggestion
  There are contributors who declared their plans to work on this initiative soon, but there is no active development.
```

File: jep/14/README.adoc [hunk 3]
```diff
@@ -83,70 +83,101 @@ Current version of the roadmap can be found here: link:https://www.jenkins.io/pr
...
+* *Preview* - This initiative is available to Jenkins users and contributors for preview.
+  It may mean a release in the experimental update center, generally-available release with proper preview notice,
+  opt-in feature flag or configuration option, Beta API, or other documented way to deliver experimental changes in Jenkins.
+* **Current** - Things being worked on presently with a known scope and design, typically specified a JEP.
```
Comment (slide) [inline jep/14/README.adoc:95]:
```suggestion
* **Current** - Things being worked on presently with a known scope and design, typically specified in a JEP.
```

File: jep/14/README.adoc [hunk 4]
```diff
@@ -178,20 +209,20 @@ In order to be displayed on the roadmap, an initiative should match the followin
 Any Jenkins contributor can submit a pull request with a suggestion for a Jenkins roadmap.
 Suggestions will be reviewed by “Roadmap maintainers”, and they will get the SIGs, sub-projects and subject matter experts involved if needed.
 
-* Each Jenkins project entity (e.g. SIG/sub-project/team) is eligible to add their roadmap entries.
-  In such cases these entries should be discussed in the entity channels and signed off by entity leaders.
+* Each Jenkins community entity (e.g. SIG/sub-project/team) is eligible to add their roadmap entries.
+  In such case initiatives should be discussed in the entity channels and signed off by the entity leader(s).
```
Comment (slide) [inline jep/14/README.adoc:213]:
```suggestion
  In such case, initiatives should be discussed in the entity channels and signed off by the entity leader(s).
```

**Comment by oleg-nenashev** [comment]
Thanks @slide ! I will merge the patch so that the JEP is up to date

**Pull request merged**